docs/metric_service_features.md

```diff
@@ -0,0 +1,118 @@
+# Metric Feature Coverage by Service
+
+## TOC
+
+- [Ad Service](#ad-service)
+- [Cart Service](#cart-service)
+- [Checkout Service](#checkout-service)
+- [Currency Service](#currency-service)
+- [Email Service](#email-service)
+- [Feature Flag Service](#feature-flag-service)
+- [Frontend Service](#frontend-service)
+- [Payment Service](#payment-service)
+- [Product Catalog Service](#product-catalog-service)
+- [Recommendation Service](#recommendation-service)
+- [Shipping Service](#shipping-service)
+
+## OpenTelemetry Feature Set by Services
+
+### Emoji Legend
+
+- Completed: :100:
+- Not Present (Yet): :construction:
+
+### [Ad Service](../src/adservice/README.md)
+
+tba
```

Comment:
Minor, normally I've seen folks putting "TBD" instead of "tba"

Reply:
In my mind it’s already decided we’ll add the info. Just not there yet 😉 but I’ll update